21torr/hosting/3.2/manifest.json

```
@@ -0,0 +1,15 @@
+{
+    "bundles": {
+        "Torr\\Hosting\\HostingBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "env": {
+        "HOSTING_TIER": "development",
+        "HOSTING_INSTALLATION": "todo-add-your-unique-installation-key-here"
```

I think either value should be commented out or have valid value.

```suggestion
        "HOSTING_INSTALLATION": ""
```